simplenode/compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       # - "-chain=test"                       # testnet
       - "-rpcallowip=0.0.0.0/0"
       - "-rpcbind=0.0.0.0"
-      - "-rpcuser=rpc"
+      - "-rpcuser=bitcoinrpc"
```

Comment:
ahh I wanted to standardize on `rpc:rpc` _everywhere_... can we change it on the other end?